.github/workflows/pull-request-labeler.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2020 Dremio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Auto Labeler
+
+on:
+  pull_request_target:
```

Comment:
trailing `.` looks odd, should we use the suggestion from the action README?
```
on:
- pull_request_target
```

Reply:
Just style from everywhere else.